plugins/woocommerce/src/Admin/API/MobileAppMagicLink.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * REST API Data countries controller.
+ *
+ * Handles requests to the /data/countries endpoint.
```

Comment:
```suggestion
 * Handles requests to the /login-qr endpoint.
```

Reply:
I've changed the base to "mobile-app/" instead since "login-qr/" doesn't make sense anymore. Thanks for catching that!